### ISSUE TITLE
minor CLI "Next steps" UX improvement

### ISSUE THREAD

**Issue by stephanietuerk** [observablehq/framework#1134]
I noticed that in using the CLI to set up a new project, the "Next steps..." box gives the user some instructions that may trip up someone who is very very new to using the command line. Namely, the instructions tell a user to `cd` to what they entered as a `rootPath`, but that `rotPath` variable includes './' at the front of it which a users doesn't actually want to include if they `cd` into that directory. 

Obviously anyone experienced with the command line can figure out what to do, but given that this product is aimed at users with a very wide range of skillsets (and that you're clearly trying to make the CLI feel nice for people!), it seems like this could potentially trip someone up. 

![image](https://github.com/observablehq/framework/assets/13357861/0209bb7f-c1d7-412d-9898-262de1cfe330)

**Comment by mbostock** [comment]
I guess this is a Windows-specific issue? On POSIX systems (namely macOS and Linux), it’s certainly valid to

```sh
cd ./hello-framework
```

and equivalent to

```sh
cd hello-framework
```

Or is this just an aesthetic issue where seeing the leading `./` introduces the question of what it means?

**Comment by mbostock** [comment]
Thanks for the suggestion! I think it makes sense to remove the `./` since we don’t need it on POSIX, and it’s actively unhelpful on Windows, so I’ve removed it in #1135. 🙏

**Comment by stephanietuerk** [comment]
Yup, seems like it's Windows specific. :) Thanks!

**Comment by mbostock** [comment]
Thank you! We’re working to overcome our POSIX assumptions and appreciate the feedback. 😅

**Comment by stephanietuerk** [comment]
UPDATE: 
I just tested this pretty thoroughly across all three Windows terminals -- bash (linux), PowerShell, and Command Prompt, and I could not reproduce. (It seemed strange to me that a path to create the directory wouldn't work to change into it and I was curious).

It IS true however that PowerShell uses `.\` instead of `./`, which is what I thought the problem was initially when you mentioned Windows. But in my testing of this, if I left the directory as ./hello-framework, and then did cd ./hello-framework, even with PowerShell I could still get there. (And the dir was created as a subdirectory of my current dir.)

Apologies for the false alarm here. My guess is that perhaps I typed something wrong in specifying the new repo location -- originally I got an error message when trying to cd. (Screenshot above was from my second time). 

Since you are interested in the Windows UX though, I'll share that if I typed something wrong, it may be due to the fact that it took me a long time to figure out how to delete the suggested "./hello-framework" under "Where to create your framework?" Neither the delete nor the arrow keys had any effect in trying to modify that text and in fact I actually quit the CLI the first time because I couldn't figure it out. I finally did figure out that I could type over top of it, but this was non-obvious because on my Windows machine (in all of the three terminals in VSCode) that text appears as solid white. When I tried this on a Mac (zsh and bash in VSCode), the text had reduced opacity/was gray, which is what I am used to for "suggested but non-mandatory" text in the command line.

![image](https://github.com/observablehq/framework/assets/13357861/27d17e1a-b0d6-4a58-906b-2edb1eaf1a7f)
![image](https://github.com/observablehq/framework/assets/13357861/bf2ef5a3-d276-4d37-991d-ad0576fff615)

In any case, given that I couldn't reproduce, I'm fine with this issue being closed. Hope the Windows info can be helpful though!
